Dockerfile

```diff
@@ -0,0 +1,39 @@
+FROM docker.io/debian:10.5-slim as base
+
+RUN apt-get update \
+  && apt-get install -y --no-install-recommends \
+     bash \
+     jq \
+     less \
+     moreutils \
+     procps \
+  && rm -rf /var/lib/apt/lists/*
+
+FROM base as downloader
+
+RUN apt-get update \
+  && apt-get install -y --no-install-recommends \
+     ca-certificates \
+     curl \
+  && rm -rf /var/lib/apt/lists/*
+
+RUN curl -sLo /tmp/kubectl "https://storage.googleapis.com/kubernetes-release/release/$(curl -s https://storage.googleapis.com/kubernetes-release/release/stable.txt)/bin/linux/amd64/kubectl" \
+  && chmod +x /tmp/kubectl
+
+RUN curl -sLo /tmp/krossa.tar.gz https://github.com/appuio/krossa/releases/download/v0.0.4/krossa_0.0.4_linux_amd64.tar.gz \
+  && mkdir /tmp/krossa \
+  && tar -xzf /tmp/krossa.tar.gz --directory /tmp/krossa \
+  && ls /tmp/krossa
+
+FROM base
+
+RUN mkdir /data \
+  && chown 1001:0 /data
+
+COPY --from=downloader /tmp/kubectl /usr/local/bin
+COPY --from=downloader /tmp/krossa/krossa /usr/local/bin
+COPY pre-backup /usr/local/bin
```

Comment:
The script doesn't really need to be called `pre-backup` anymore. `object-backup` or `k8s-backup` might be more appropriate.

Reply:
Renamed do `dump-objects`.